packages/react-dom/src/events/DOMModernPluginEventSystem.js

```diff
@@ -198,11 +209,22 @@ export function listenToTopLevelEvent(
   listenerMap: ElementListenerMap,
   passive?: boolean,
   priority?: EventPriority,
+  capture?: boolean,
 ): void {
-  const listenerEntry = listenerMap.get(topLevelType);
+  // If we explicitly define capture, then these are for EventTarget objects,
+  // rather than React managed DOM elements. So we need to ensure we separate
+  // capture and non-capture events (for React managed DOM nodes we only use
+  // one or the other, never both).
```

Comment:
> "*for React managed DOM nodes we only use one or the other, never both*"

I'm not sure I understand this comment. It sounds like it's saying that you can't attach both a capture *and* bubble event to a React managed DOM node, but looking at the implementation, that doesn't seem correct.

Maybe you could change the wording to something clearer?

Reply:
That is correct, we never attach a catpture and bubble event listener to React managed DOM nodes. We have a Set that determines which one we do use, based on the event name, and we use one one and we emulate the effect of bubble/capture when we dispatch a two phase event. I'll update the comment to include that info.